onnx/defs/math/defs.cc

```diff
@@ -463,3 +463,31 @@ OPERATOR_SCHEMA(MatMul)
     .SetDoc(R"DOC(
 Matrix product that behaves like numpy.matmul: https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.matmul.html
 )DOC");
+
+OPERATOR_SCHEMA(Identity)
+    .NumInputs(1)
+    .NumOutputs(1)
+    .SetDoc("Identity operator")
+    .Input(0, "input", "Input tensor", "T")
+    .Output(
+        0,
+        "output",
+        "Tensor to copy input into. Can be in-place",
+        "T")
+    .TypeConstraint("T", { "tensor(float16)", "tensor(float)", "tensor(double)" },
+        "Constrain input and output types to float tensors.");
+
+OPERATOR_SCHEMA(Linear)
```

Comment:
The name is bad as it can be confused with the fully connected layer. Maybe Scale or Affine? 

Also I wonder whether we should just use add and multiply instead if that's not perf-critical